lib/libutee/include/pta_device.h

```diff
@@ -0,0 +1,41 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) 2019, Linaro Limited
+ */
+
+#ifndef __PTA_DEVICE_H
+#define __PTA_DEVICE_H
+
+#define PTA_DEVICE_UUID { 0x7011a688, 0xddde, 0x4053, \
+		{ 0xa5, 0xa9, 0x7b, 0x3c, 0x4d, 0xdf, 0x13, 0xb8 } }
+
+/*
+ * PTA_CMD_GET_NUM_DEVICES - Get number of TEE bus devices
+ *
+ * param[0] (out value) - value.a: Number of devices present
```

Comment:
Maybe prefer to **pta_invoke_test.h** style:
```C
/*
 * Get number of TEE bus device
 *
 * [out]     value[0].a        Number of devices present
 */
#define PTA_CMD_GET_NUM_DEVICES		0x0

/*
 * Get device UUIDs
 *
 * [out]     memref[0]        Array of device UUIDs
 */
#define PTA_CMD_GET_DEVICES		0x1
```

Reply:
Along with this, I think adding return codes here would bring more clarity to the client/user.